src/pages/registration/contributors_tab/components/ContributorRow.tsx

```diff
@@ -84,28 +72,24 @@ export const ContributorRow = ({
             }}
             onBlur={handleOnMoveContributor}
           />
-          {!isLastElement && (
-            <Tooltip title={t('common.move_down')}>
-              <IconButton
-                size="small"
-                disabled={disableChannelClaimsFields}
-                sx={{ minWidth: 'auto', height: 'fit-content', marginTop: '0.6rem' }}
-                onClick={() => onMoveContributor(contributor.sequence + 1, contributor.sequence)}>
-                <ArrowDownwardIcon color="primary" />
-              </IconButton>
-            </Tooltip>
-          )}
-          {contributor.sequence !== 1 && (
-            <Tooltip title={t('common.move_up')}>
-              <IconButton
-                size="small"
-                disabled={disableChannelClaimsFields}
-                sx={{ minWidth: 'auto', height: 'fit-content', marginTop: '0.6rem' }}
-                onClick={() => onMoveContributor(contributor.sequence - 1, contributor.sequence)}>
-                <ArrowUpwardIcon color="primary" />
-              </IconButton>
-            </Tooltip>
-          )}
+          <MoveArrowButton
+            orientation="up"
+            index={contributorIndex}
+            disabled={disableChannelClaimsFields || contributor.sequence === 1}
+            sx={{ minWidth: 'auto', height: 'fit-content', marginTop: '0.6rem' }}
```

Comment:
Virker som dette er lagt inn for at knappen skal bli rund? I så fall er det kanskje bedre å heller bare sette `alignItems: 'center'` på flex parent, for å unngå litt mystisk styling jeg hvertfall ikke skjønte helt hva var uten å prøve å ta koden inn og ut og inspisere hva som skjer 😅

Reply:
fjerna de bare, da det ikke hadde noen stor innvirkning